src/constant_texts.c

```diff
@@ -265,6 +265,8 @@ const char *ui_text_start_auth_from_CySync =
 const char *ui_text_invalid_card_contact_cypherock =
     "Authentication failed Contact Cypherock";
 const char *ui_text_wrong_card_sequence = "Wrong card sequence";
+const char *ui_text_tap_another_card_already_tapped =
+    "Tap another card\n Already tapped";
```

Comment:
there was a comment to make this as "Tap another card". verify with @gauravmalviya-cypher